presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestBingTileOperators.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial;
+
+import com.facebook.presto.operator.scalar.AbstractTestFunctions;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+
+public class TestBingTileOperators
+        extends AbstractTestFunctions
+{
+    @Test
+    public void testApproxDistinct()
```

Comment:
The other operators, distinct-from and equals, are tested in TestBingTileFunctions (see testDistinctFrom and testEquals). For consistency, consider moving this test there or moving existing tests here.